packages/react-components/src/Concepts/Slots.stories.mdx

```diff
@@ -62,11 +62,11 @@ It also has an `icon` slot that only renders when there is neither an image nor
 A property of a component can be identified as a slot by its type.
 
 - IntrinsicShorthandProps: The props for a slot that supports one or more HTML element types, such as 'div'.
-- ObjectShorthandProps: The props for a slot that additionally supports a JSX component or component props object.
+- ComponentSlotProps: The props for a slot that additionally supports a JSX component or component props object.
 - ShorthandRenderFunction: A render function for a slot.
 
-Notice these types contain the term 'shorthand'.
-Shorthand means the slot props can accept 'shorthand' data such as primitive values and props objects.
+In our composition system, slot props can accept 'shorthand' data such as primitive values and props objects, that they
+use to determine the overall.
```

Comment:
This is in the other PR. See my comment there. https://github.com/microsoft/fluentui/pull/20891#discussion_r760715363